.circleci/config.yml

```diff
@@ -52,8 +52,6 @@ jobs:
       # restore_cache step above can override the go-filecoin code with cached code
       - git_fetch_all_tags
       - checkout
-      - restore_cache:
```

Comment:
BLOCKING: See comment below. Removing this `restore_cache` will cause `"deps"` to download/generate Groth parameters.